detekt-rules-empty/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/empty/EmptyKotlinFileSpec.kt

```diff
@@ -0,0 +1,36 @@
+package io.gitlab.arturbosch.detekt.rules.empty
+
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.test.assertThat
+import io.gitlab.arturbosch.detekt.test.compileAndLint
+import org.junit.jupiter.api.Test
+
+class EmptyKotlinFileSpec {
+    private val subject = EmptyKotlinFile(Config.empty)
+
+    private val codeWithPackageStatement = """
+            package my.package
```

Comment:
`package` is a reserved word, and for that reason this code doesn't compile

```suggestion
            package my.packagee
```